packages/core/test/insured-bridge/BridgeDepositBox.js

```diff
@@ -53,96 +60,205 @@ describe("BridgeDepositBox", () => {
 
-    it("Reverts on non-whitelisted token", async () => {
-      const l2Token_nonWhitelisted = await Token.new("L2 Wrapped Ether", "WETH", 18).send({ from: deployer });
-      await l2Token_nonWhitelisted.methods.addMember(1, deployer).send({ from: deployer });
-      await l2Token_nonWhitelisted.methods.mint(user1, toWei("100")).send({ from: deployer });
-
-      await l2Token_nonWhitelisted.methods.approve(depositBox.options.address, toWei("100")).send({ from: user1 });
-
-      const quoteTimestamp = Number(await timer.methods.getCurrentTime().call()) + quoteTimestampOffset;
-      assert(
-        await didContractThrow(
-          depositBox.methods
-            .deposit(
-              user1,
-              l2Token_nonWhitelisted.options.address,
-              depositAmount,
-              slowRelayFeePct,
-              instantRelayFeePct,
-              quoteTimestamp
-            )
-            .send({ from: user1 })
-        )
-      );
+        assert.equal((await l2Token.methods.balanceOf(depositBox.options.address).call()).toString(), "0");
+
+        const quoteTimestamp = Number(await timer.methods.getCurrentTime().call()) - quoteTimestampOffset;
+        const tx = await depositBox.methods
+          .deposit(user1, l2Token.options.address, depositAmount, slowRelayFeePct, instantRelayFeePct, quoteTimestamp)
+          .send({ from: user1 });
+
+        assert.equal((await l2Token.methods.balanceOf(depositBox.options.address).call()).toString(), depositAmount);
+
+        await assertEventEmitted(tx, depositBox, "FundsDeposited", (ev) => {
+          return (
+            ev.chainId == "10" &&
+            ev.depositId == "0" &&
+            ev.l1Recipient == user1 &&
+            ev.l2Sender == user1 &&
+            ev.l1Token == l1TokenAddress &&
+            ev.l2Token == l2Token.options.address &&
+            ev.amount == depositAmount &&
+            ev.slowRelayFeePct == slowRelayFeePct &&
+            ev.instantRelayFeePct == instantRelayFeePct &&
+            ev.quoteTimestamp == quoteTimestamp
+          );
+        });
+
+        assert.equal(await depositBox.methods.numberOfDeposits().call(), "1"); // Deposit index should increment to 1.
+      });
     });
-    it("Reverts if deposits disabled", async () => {
-      // Disable deposits
-      await depositBox.methods.setEnableDeposits(l2Token.options.address, false).send({ from: bridgeAdmin });
-
-      // Try to deposit and check it reverts.
-      await l2Token.methods.approve(depositBox.options.address, toWei("100")).send({ from: user1 });
-      const quoteTimestamp = Number(await timer.methods.getCurrentTime().call()) + quoteTimestampOffset;
-      assert(
-        await didContractThrow(
-          depositBox.methods
-            .deposit(user1, l2Token.options.address, depositAmount, slowRelayFeePct, instantRelayFeePct, quoteTimestamp)
-            .send({ from: user1 })
-        )
-      );
+
+    describe("Eth deposit logic", () => {
```

Comment:
What's the gas difference in this `describe` block vs the `ERC20 deposit` `describe` block?